deploy/ironic_bmo_configmap.env

```diff
@@ -6,3 +6,4 @@ DEPLOY_RAMDISK_URL=http://172.22.0.2:6180/images/ironic-python-agent.initramfs
 IRONIC_ENDPOINT=http://172.22.0.2:6385/v1/
 IRONIC_INSPECTOR_ENDPOINT=http://172.22.0.2:5050/v1/
 CACHEURL=http://172.22.0.1/images
+OPERATOR_CONCURRENCY=3
```

Comment:
If this is the default anyway, is there any reason to specify it explicitly?
I ask because it's a lot of places to change if we choose a different default later.

Reply:
You're right. Its probably not needed. Since we'll have it specified in the controller.